doc/src/sgml/ref/create_subscription.sgml

```diff
@@ -611,7 +607,7 @@ falseの場合、レプリケーションワーカーは各テーブルでその
           replication can be resumed from the new primary after failover.
           The default is <literal>false</literal>.
 -->
-《機械翻訳》フェイルオーバー後に新しいプライマリから論理レプリケーションを再開できるように、サブスクリプションに関連付けられたレプリケーションスロットをスタンバイに同期できるようにするかどうかを指定します。
+フェイルオーバー後に新しいプライマリから論理レプリケーションを再開できるように、サブスクリプションに関連付けられたレプリケーションスロットが物理スタンバイと同期できるようにするかどうかを指定します。
```

Comment:
「物理スタンバイと同期」の「物理」はどの部分の訳になるでしょうか。
私は見つけられずでして。the standbysが物理スタンバイ・・ではない気もしています。

Reply:
`to the standbys so that...`で触れられているstandbysが物理スタンバイだと思っています。
ただ原文にない表現ですので削除しました。

`CREATE SUBSCRIPTION`の`failover`オプションはそのまま論理レプリケーションスロットの`failover`に渡されて、スロット同期機能の対象にするかどうかを決めることができます。スロット同期はストリーミングレプリケーション間の話なので、意味合いとしては物理スタンバイとなるはずです。

Reply:
すみません。理解が追いつきました。確かに機能的にfailover先は物理レプリケーションですね。
ただ原文にはないので修正いただいた内容でお願いします。対応ありがとうございます。